auction-deploy/auction_deploy/core.py

```diff
@@ -0,0 +1,132 @@
+import json
+import pkg_resources
+from typing import Dict, NamedTuple
+from collections import namedtuple
+
+from eth_keyfile import extract_key_from_keyfile
+from deploy_tools.deploy import send_function_call_transaction, deploy_compiled_contract
+
+
+class AuctionOptions(NamedTuple):
+    start_price: int
+    auction_duration: int
+    number_of_participants: int
+    release_block_number: int
+
+
+DeployedAuctionContracts = namedtuple("DeployedContracts", "locker slasher auction")
+
+
+def load_contracts_json() -> Dict:
+    resource_package = __name__
+    stream = pkg_resources.resource_stream(resource_package, "contracts.json")
```

Comment:
from reading the setuptools documentation,  I think you should be able to use `pkg_resources.resource_string`. Please try to use that.

Reply:
done